src/components/book-form.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../actions/index';
```

Comment:
please make it a practice to import just what you need using de structuring instead of the whole thing, it could be large.